Meter.c

```diff
@@ -86,6 +108,9 @@ void Meter_delete(Object* cast) {
    if (Meter_doneFn(this)) {
       Meter_done(this);
    }
+   if (this->drawData != NULL) {
+      free(this->drawData->cells);
+   }
```

Comment:
Didn't expect this style. I prefer writing conservatively and explicitly checking pointer against NULL, rather than depending on implicit cast from pointer to boolean.

Reply:
Both with and without the explicit check is fine, but the codebase favours the variant without.